pwa/src/components/objectEntities/ObjectEntitiesTable.tsx

```diff
@@ -161,7 +161,16 @@ const ObjectEntitiesTable: React.FC<ObjectEntitiesTableProps> = ({ entityId }) =
             <Modal
               title={`Create a new ${entity?.name} object`}
               id="objectModal"
-              body={() => <>{FormIO && FormIO}</>}
+              body={() => (
+                <>
+                  { FormIO && formIOSchema.components && FormIO }
```

Comment:
I think we could make sure (or maybe already are making sure..) that `formIOSchema.components` is available, so we don't have to do this check in the render function.

1. Can `FormIO` exist without it?
2. We're checking on line `42` for `formIOSchema` before setting `FormIO`, this check might already be enough, if not, we might want to change this check to `formIOSchema.components` and
3. The check could then be `{ FormIO && FormIO }` or maybe just `{ FormIO }` without a check.